Formula/unum.rb

```diff
@@ -1,8 +1,14 @@
 class Unum < Formula
   desc "Interconvert numbers, Unicode, and HTML/XHTML entities"
   homepage "https://www.fourmilab.ch/webtools/unum/"
-  url "https://www.fourmilab.ch/webtools/unum/prior-releases/3.2/unum.tar.gz"
-  sha256 "d290070f4be54addacac7318dfb2c0bfde71690bba51f99ecf64671b71801d2a"
+  url "https://www.fourmilab.ch/webtools/unum/prior-releases/3.3/unum.tar.gz"
+  sha256 "544dd1172665c38237bbeb2d4d820329064e6a78373486b92fbd498714dc2e91"
+  license "Artistic-1.0-Perl"
+
+  livecheck do
+    url :homepage
+    regex(%r{This is <strong>unum</strong> version (\d+(?:\.\d+)+)}i)
```

Comment:
We prefer checking the location of the stable archive instead, so this might be better:
```suggestion
    url "https://www.fourmilab.ch/webtools/unum/prior-releases/"
    regex(%r{href=["']?v?(\d+(?:\.\d+)+)/?["' >]}i)
```